packages/onramp-kit/src/types.ts

```diff
@@ -1,28 +1,28 @@
-import { StripeAdapter } from './packs/stripe/StripeAdapter'
+import { StripePack } from './packs/stripe/StripePack'
 import {
   StripeSession,
   StripeEvent,
   StripeEventListener,
   StripeOpenOptions
 } from './packs/stripe/types'
 
-// The new adapters must implement this interface
-export interface SafeOnRampAdapter<TAdapter> {
+// The new packs must implement this interface
+export interface SafeOnRampPack<TPack> {
   init(): Promise<void>
-  open(options?: SafeOnRampOpenOptions<TAdapter>): Promise<SafeOnRampOpenResponse<TAdapter>>
+  open(options?: SafeOnRampOpenOptions<TPack>): Promise<SafeOnRampOpenResponse<TPack>>
   close(): Promise<void>
-  subscribe(event: SafeOnRampEvent<TAdapter>, handler: SafeOnRampEventListener<TAdapter>): void
-  unsubscribe(event: SafeOnRampEvent<TAdapter>, handler: SafeOnRampEventListener<TAdapter>): void
+  subscribe(event: SafeOnRampEvent<TPack>, handler: SafeOnRampEventListener<TPack>): void
+  unsubscribe(event: SafeOnRampEvent<TPack>, handler: SafeOnRampEventListener<TPack>): void
 }
 
-// When creating new adapters these types should be updated:
+// When creating new packs these types should be updated:
 // e.g.:
 // export type SafeOnRampOpenOptions<T> =
-//    T extends StripeAdapter ? StripeOpenOptions :
-//    T extends FooAdapter ? FooOpenOptions :
-//    T extends BarAdapter ? BarOpenOptions :
+//    T extends StripePack ? StripeOpenOptions :
+//    T extends FooPack ? FooOpenOptions :
+//    T extends BarPack ? BarOpenOptions :
```

Comment:
According to lines 25-28 it should be renamed here as well
```suggestion
//    TPack extends StripePack ? StripeOpenOptions :
//    TPack extends FooPack ? FooOpenOptions :
//    TPack extends BarPack ? BarOpenOptions :
```